code/about.html

```diff
@@ -0,0 +1,36 @@
+<html>
+
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Project Name</title>
+    <!-- dont forget to add a css file and link it here! -->
+    <link   rel="stylesheet"    type="text/css" href="style.css"  
+    <!-- If you're using google fonts, that should be added here -->
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Advent+Pro:wght@100;200;300;400;500;600;700&family=Amatic+SC:wght@400;700&display=swap" rel="stylesheet">
+  </head>
+  <body id="about">
+
+    <menu>
+      <a href="./index.html">Home</a>
+      <a href="./work.html">Work</a>
+      <a href="./contact.html">Contact</a>
+    </menu>
+
+    <div class="aboutme">
+      <h1 > About me</h1>
+      <img id="portrait" src="./img/Portrait.png"  > <img>
+    </div>
+    <div class="textabschnitt">
+      <p> I studied Animation at the University of Art and Design in Lucerne with the main focus on CGI Animation, Concept Art and Storyboarding. 
+        At the Moment i'm doing an internship at an advertisment studio in Zurich.
+        My passion lies in drawing and painting, as well as animating. 
+        At the moment i'm learning how to do Frontend design.
+       </p>
+      </div>
```

Comment:
Be careful with **indentation** at all times. The rule to follow here is that if there's a new block, then you indent one more level. At Technigo, we use 2 spaces for indentation. 🔍